### DISCUSSION TITLE
Switching off edit warnings

### DISCUSSION THREAD

**Discussion by Britmetal** [sn4k3/UVtools#174] [Q&A] [answered]
Like the title says, is there a way to switch off the warnings when editing layers?

**Comment by sn4k3** [comment, marked as answer]
You mean this:
![image](https://user-images.githubusercontent.com/113281/113036188-e0077980-918b-11eb-999b-17e45247856b.png)
?

**Reply by Britmetal** [reply]
Yes and 
![error](https://user-images.githubusercontent.com/39800614/113039962-28c13180-9190-11eb-81e3-dd747b3f91ca.jpg)

I been fixing some very nasty stls which have required lof of work

**Reply by sn4k3** [reply]
Wow that's a lot of issues you are removing... 
But no, there are no way to skip that dialog as it's always good pratice prompt before removal, eg: acidental click or delete.
As quick way you can use shortcuts, like click remove and press ENTER on keyboard OR press  DELETE and ENTER on keyboard

**Reply by Britmetal** [reply]
Thanks for answering, not my files, a friend wanted to know why a couple of files he has failed to print. rather unwisely I offered to help.
